pkg/fakerp/vault.go

```diff
@@ -94,7 +104,20 @@ func (vm *vaultManager) writeTLSCertsToVault(ctx context.Context, cs *api.OpenSh
 		if err != nil {
 			return err
 		}
-		err = vault.ImportCertificate(ctx, vm.kvc, vaultURL, cert.vaultKeyName, newkey, newcert)
+
+		// This chain should follow Certificate chain practices, where oder is:
```

Comment:
order

Reply:
need more coffee... was looking to this for 10 minutes and thinking how this order needs to change :D